cartesian_space_controller:
  ros__parameters:
    type: tsid_controllers/CartesianSpaceController
    update_rate: 25
  
    local_frame: false
    
    joint_state_names:
      - arm_right_1_joint
      - arm_right_2_joint
      - arm_right_3_joint
      - arm_right_4_joint
      - arm_right_5_joint
      - arm_right_6_joint
      - arm_right_7_joint
      - torso_lift_joint

    joint_command_names:
      - arm_right_1_joint
      - arm_right_2_joint
      - arm_right_3_joint
      - arm_right_4_joint
      - arm_right_5_joint
      - arm_right_6_joint
      - arm_right_7_joint

    interface_name: position

    ee_names:
      - arm_right_7_link

    velocity_scaling: 1.0

    posture_gain: 1.0

    cartesian_gain:
         arm_right_7_link:
          #kp_x: 1500.0
          #kp_y: 1500.0
          #kp_z: 1500.0
          #kp_roll: 700.0
          #kp_pitch: 700.0
          #kp_yaw: 700.0
          kp_x: 500.0
          kp_y: 500.0
          kp_z: 1000.0
          kp_roll: 70.0
          kp_pitch: 70.0
          kp_yaw: 70.0


